### PR TITLE
windex_v6:  Add some HE v100 opcodes.

### PR DESCRIPTION
**Pull request by LittleToonCat**
This PR adds some of the opcodes for HE v100 (v101) games.  This has been tested with Backyard Basketball (which is v101), and it fully decompiles without issues.  I haven't tested it with other games yet, since I'll be focusing on Basketball in particular.

### CODE REVIEW

File: src/nutcracker/sputm/windex_v6.py [hunk 1]
```diff
@@ -43,7 +43,10 @@ def __repr__(self):
             # NOTE: in HE v71 games, they put the "/" char as a
             # WordValue.
             assert isinstance(self.orig, (ByteValue, WordValue))
-            return f"'{chr(self.num)}'"
+            if self.num < 128:
+                # ASCII Char
+                return f"'{chr(self.num)}'"
+            return f"'\\{ord(chr(self.num)):02x}'".upper()
```
Comment (BLooperZ) [inline src/nutcracker/sputm/windex_v6.py]: Thank you, can you please elaborate on this part?
which characters specifically are used in this game script that need to be escaped?
overall, I'd rather pass the character as is as long as it is printable

Comment (LittleToonCat) [reply src/nutcracker/sputm/windex_v6.py]: Basketball used a `o71_getCharIndexInString` call with `'\xFF' (255)` as its value, which is not an ASCII character, causing an error when attempting to print it out.

```UnicodeEncodeError: 'ascii' codec can't encode character '\xff' in position 55: ordinal not in range(128)```

Comment (BLooperZ) [reply src/nutcracker/sputm/windex_v6.py]: Thank you, can be rewritten as `f"'\\x{ord(chr(self.num)):02X}'"` (missing x after escape, and using uppercase X as modifier to allow it)

Comment (LittleToonCat) [reply src/nutcracker/sputm/windex_v6.py]: Done in b1cd7742ef004b91840d254a443055a7b49e8a6e

**Comment by BLooperZ** [comment]
Thank you, merging

**Pull request merged**